CHANGELOG.md

```diff
@@ -6,7 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Latest](https://github.com/NordicSemiconductor/svada)
 
-## [v2.0.0](https://github.com/NordicSemiconductor/svada/tree/v2.0.0)
+## [v2.0.1](https://github.com/NordicSemiconductor/svada/tree/v2.0.1)
+
+### Added
+* Minimum version requirements for all dependencies.
+
+## [v2.0.0](https://github.com/NordicSemiconductor/svada/tree/2.0.0)
```

Comment:
The tag that was added for this version is called `2.0.0`, so I changed the link here. Maybe it would be better to add a tag called `v2.0.0` instead.

Reply:
I would say that its better to create a new tag. The old tag would still work as well

Reply:
Ok, will revert this particular change.

Reply:
fixed